src/style.css

```diff
@@ -1,3 +1,159 @@
-body {
+/* #region root */
+
+:root {
+  --main-accent: #060b35;
+  --secondary: #616070;
+  --blue-accent: #00acdc;
+  --gray-elements: #f3f3f3;
+  --background-main: #fff;
+  --starsCount: 4;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  color: var(--main-accent);
+}
+
+body,
+p,
+h2,
+h3 {
   margin: 0;
+  padding: 0;
+}
+
+/* #endregion */
+/* #region card main */
+
+.card {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+  height: 406px;
+  width: 198px;
+  border-radius: 5px;
+  background: var(--background-main);
+  border: 1px solid var(--gray-elements);
+}
+
+.card__container {
+  padding: 0 16px 16px;
+  width: fit-content;
+}
+/* #endregion */
+/* #region card image */
+
+.card__image {
+  height: 134px;
+  width: 160px;
+  margin: 32px 19px 40px;
+}
+/* #endregion */
+/* #region title */
+.card__container-title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-accent);
+  margin-bottom: 4px;
+}
+/* #endregion */
+/* #region product code */
+.card__container-product-code {
+  display: block;
+  margin-bottom: 16px;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+}
+/*#endregion*/
+/*#region rating*/
+.card__container-rating {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-position: left;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg);
+}
+
+.card__container-rating-reviews {
+  display: flex;
+  justify-content: end;
+  align-items: center;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-accent);
+}
+/*#endregion*/
+/*#region price*/
+.card__container-price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  align-items: center;
+}
+
+.card__container-price-title {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--secondary);
```

Comment:
move all the duplicated styles to variables, like font-size, line-line-height, background, etc.